.github/workflows/ci.yml

```diff
@@ -10,6 +13,8 @@ on: # yamllint disable-line rule:truthy rule:comments
   pull_request: ~
 
 env:
+  # TODO Fix to be repository name
```

Comment:
As we go to 2.0, this makes sense to fix.

Reply:
Fixed